### ISSUE TITLE
Resnet-152 loss explode after lr decay

### ISSUE THREAD

**Issue by WonderAndMaps** [tensorflow/tpu#304]
I am training a Resnet-152 with `tpu/models/official/resnet/resnet_main.py` on custom data. I am seeing the loss going up right after the learning rate decay (*0.1) and after a while it blew up, which is counter intuition. (see screen shot)

I am sure the data is fine since it works on other models (inceptions, amoeba). I am not sure if it is Resnet or Resnet-152. I will try Resnet-50 and post updates. 

Anyone seen this before or got suggestions? 

![image](https://user-images.githubusercontent.com/40251504/53981477-765d3400-40c7-11e9-90b3-8655b0050135.png)

**Comment by louisquinn** [comment]
Hey @WonderAndMaps I am getting the exact same issue with Resnet-101, see my attached screenshot. 
Did you work out any improvements? I'm basically using the defaults for initial learning rate, decay etc.

![Screenshot from 2020-08-01 18-02-01](https://user-images.githubusercontent.com/5274490/89097305-3667c100-d421-11ea-99e0-2411f5bddfa9.png)

**Comment by WonderAndMaps** [comment]
I couldn’t find a solution. Ended up trying other models.

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: louisquinn <notifications@github.com>
发送时间: Saturday, August 1, 2020 1:05:15 AM
收件人: tensorflow/tpu <tpu@noreply.github.com>
抄送: Yang Fu <yang.fu@clobotics.com>; Mention <mention@noreply.github.com>
主题: Re: [tensorflow/tpu] Resnet-152 loss exploded after lr decay (#304)


Hey @WonderAndMaps<https://github.com/WonderAndMaps> I am getting the exact same issue with Resnet-101, see my attached screenshot.
Did you work out any improvements? I'm basically using the defaults for initial learning rate, decay etc.

[Screenshot from 2020-08-01 18-02-01]<https://user-images.githubusercontent.com/5274490/89097305-3667c100-d421-11ea-99e0-2411f5bddfa9.png>

―

**Comment by louisquinn** [comment]
@WonderAndMaps cool, thanks for your response. 

Yeah I am currently trying again with the `models/official/detection` package (you can train classifier models with it). It also has more flexibility with hyperparams/configs and many more models implemented. Fine-tuning is available too.
